packages/context/src/interceptor.ts

```diff
@@ -104,6 +104,24 @@ export class InvocationContext extends Context {
     });
   }
 
+  get targetClass() {
+    return typeof this.target === 'function'
+      ? this.target
+      : this.target.constructor;
+  }
+
+  get targetName() {
+    return DecoratorFactory.getTargetName(this.target, this.methodName);
+  }
+
+  get description() {
+    return `InvocationContext ${this.name}: ${this.targetName}`;
+  }
+
+  toString() {
+    return this.description;
+  }
```

Comment:
Let's contribute these improvements to `@loopback/context` in a new pull request, make sure to include automated tests.

Reply:
See https://github.com/strongloop/loopback-next/pull/2866